database/migration/migrateData.ts

```diff
@@ -206,11 +212,13 @@ const migrateData = async () => {
       RETURN NEW; END;
       $$ LANGUAGE plpgsql;`
     )
-  
-  // New field description returned in permissions_all
-  console.log(' - Add permission_all VIEW field: description');
-  
-  await DB.changeSchema(`CREATE OR UPDATE VIEW permissions_all AS (
```

Comment:
This wasn't correct. Should be "CREATE OR REPLACE VIEW". But that didn't work either, as adding columns in the middle of view wasn't allowed, so have to do "DROP VIEW" first.

Also -- this obviously hadn't been tested since it reported an obvious error when running `yarn migrate`. Need to remember to always thoroughly test migration code!

Reply:
Yes... My bad, I did this without being able to do a proper check. I had it changed on the schema_init for the snapshot, so probably why I didn't see the error coming up. I hadn't tested with `yarn migrate` though. It would be better to improve the READ-ME page to list all these cases for development, since it's an open-source project we don't want other people running into the same problems.

Reply:
Will do!

Reply:
Done! ce996962